<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QWidget" name="GameWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 130, 0);</string>
  </property>
  <widget class="QLabel" name="lbPlayerCard1">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>445</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string>CARD</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPlayerCard2">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>445</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string>CARD</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbCompCard2">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>5</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string>CARD</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbCompCard1">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>5</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string>CARD</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTableCard2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string>CARD</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTableCard1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string>CARD</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTableCard4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string>CARD</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTableCard3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string>CARD</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTableCard5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>250</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string>CARD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCall">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>515</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Check/Call</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbFold">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>515</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Fold</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRaise">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>550</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Raise</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>550</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNextDeal">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>300</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Next Deal</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPlAmountInfo">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>480</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Your Amount:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPlAmount">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>480</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbCompAmount">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbCompAmountInfo">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Comp Amount:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbWinner">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Winner is:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbCurBetInfo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current bet:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbCurBet">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbCurBankInfo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current bank:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbCurBank">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
